docs/en/observability/apm/otel-direct.asciidoc

```diff
@@ -1,10 +1,17 @@
 [[apm-open-telemetry-direct]]
-=== OpenTelemetry native support
+=== Upstream OpenTelemetry SDKs and Collectors
```

Comment:
We use a few different terms:

Landing page: Upstream OpenTelemetry language SDK or Collector
This page heading: Upstream OpenTelemetry SDKs and Collectors
This page content: OpenTelemetry collector, OpenTelemetry agent

Should we change the page content from OpenTelemetry agent to OpenTelemetry SDK?
Should we rearrange the order of the SDK section and the Collector section to match the order in the title?